### PR TITLE
Add Organization filter to shared public dashboards

### PR DESCRIPTION
**Pull request by QuentinBisson**
This PR

- adds organization filter to shared public dashboards towards https://github.com/giantswarm/thg/issues/110

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

Tested on Gauss

### CODE REVIEW

File: helm/dashboards/dashboards/shared/public/dns.json [hunk 1]
```diff
@@ -940,6 +940,35 @@
...
         "useTags": false
+      },
+      {
+        "allValue": null,
+        "current": {
+          "text": "All",
+          "value": "$__all"
+        },
+        "datasource": "$datasource",
+        "definition": "label_values(kubernetes_build_info{cluster_type=~\"workload_cluster\", cluster_id=\"$cluster\"}, organization)",
```
Comment (hervenicol) [inline helm/dashboards/dashboards/shared/public/dns.json:951]: This file has an 'organization` variable, but the graphs don't filter by organization; the variable seems unused.

Comment (QuentinBisson) [reply helm/dashboards/dashboards/shared/public/dns.json]: Thanks, I forgot to update the queries

**Comment by QuentinBisson** [comment]
Fixed the DNS dashboard, changed the filter order and added the organization filter to the aws public dashboard

**Review by hervenicol — APPROVED** [review]
LGTM if tested.

**Comment by QuentinBisson** [comment]
It was tested on Gauss :)

**Pull request merged**